.github/workflows/pkg.yml

```diff
@@ -0,0 +1,121 @@
+name: pkg releases
+on:
+  push:
+    branches-ignore:
+      - latest
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+          cache: "yarn"
+
+      - name: Tags
+        id: tags
+        uses: livepeer/action-gh-release-tags@v0
+
+      - name: yarn install
+        run: |
+          yarn install --frozen-lockfile
+
+      - name: build pkgs
+        # I want to do --parallel here, but the binary downloads conflict with each other
+        run: |
+          yarn run lerna-run pkg
+
+      - name: move pkgs
+        run: |
+          mkdir -p ./pkg-releases
+          for package in api www; do
+            for arch in linux-arm64 linux-x64 macos-arm64 macos-x64 win-x64; do
+              inName="$package-$arch"
+              outName="livepeer-$package"
+              archiveName="livepeer-$(echo $inName | sed s/x64/amd64/ | sed s/win/windows/ | sed s/macos/darwin/).tar.gz"
```

Comment:
can be shortened to:

    archiveName="livepeer-$(echo $inName | sed -e 's/x64/amd64/;s/win/windows/;s/macos/darwin/').tar.gz"

Reply:
Nice! Applied